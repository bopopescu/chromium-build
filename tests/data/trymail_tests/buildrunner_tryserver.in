{
  'build_step_props': {
    'blamelist': [u'iannucci@chromium.org'],
    'branch': None,
    'buildername': 'win_rel',
    'buildnumber': 102110,
    'clobber': True,
    'got_nacl_revision': 10619,
    'got_revision': 178097,
    'got_v8_revision': 13451,
    'got_webkit_revision': 140218,
    'gtest_filter': None,
    'issue': 11941002,
    'mastername': 'tryserver.chromium',
    'patchset': 33001,
    'parent_buildername': None,
    'parent_buildnumber': None,
    'primary_repo': '',
    'project': 'chrome',
    'repository': '',
    'requester': 'doesnt-exist@chromium.org',
    'revision': 'HEAD',
    'rietveld': 'https://chromiumcodereview.appspot.com',
    'root': 'src',
    'scheduler': 'try_job_rietveld',
    'slavename': 'vm355-m4',
    'swarm_tests': '',
    'testfilter': [u'interactive_ui_tests', u'sync_integration_tests'],
    'try_job_key': 'ahdzfmNocm9taXVtY29kZXJldmlldy1ocnIyCxIFSXNzdWUYiunYBQwLEghQYXRjaFNldBjpgQIMCxIMVHJ5Sm9iUmVzdWx0GOHXAgw',
    'warnings-count': 156,
    'workdir': 'E:\b\build\slave/win',
  },
  'build_step': {
    'getSourceStamp().revision': 'HEAD',
    'getSourceStamp().timestamp': 'XXXXXX',
    'getSourceStamp().got_revision': 'XXXXXX',
    'getReason()': '11941002-33001: Rebase',
    'getNumber()': 102110,
    'getResults()': results.SUCCESS,
    'getSlavename()': 'vm355-m4',
  },
  'builder': {
    'getName()': 'win_rel',
  },
  'steps': [
    step('svnkill'),
    step('update_scripts'),
    step('taskkill'),
    step('update', ['r178097', 'webkit r140218', 'nacl r10619', 'v8 r13451']),
    step('apply_issue'),
    step('apply_issue_buildrunner_ignore', hidden=True),
    step('runhooks', ['gclient hooks'], skip_name=True),
    step('cleanup_temp'),
    step('compile', result=results.WARNINGS),
    step('manifest_to_hashes', result=results.SKIPPED),
    step('trigger_swarm_triggered', result=results.SKIPPED),
    step('start_crash_handler'),
    step('interactive_ui_tests'),
    step('check_deps2git', result=results.SKIPPED),
    step('check_deps2submodules', result=results.SKIPPED),
    step('check_deps', result=results.SKIPPED),
    step('base_unittests', result=results.SKIPPED),
    step('cacheinvalidation_unittests', result=results.SKIPPED),
    step('cc_unittests', result=results.SKIPPED),
    step('chromedriver_unittests', result=results.SKIPPED),
    step('components_unittests', result=results.SKIPPED),
    step('courgette_unittests', result=results.SKIPPED),
    step('crypto_unittests', result=results.SKIPPED),
    step('url_unittests', result=results.SKIPPED),
    step('url_unittests_buildrunner_ignore', hidden=True),
    step('gpu_unittests', result=results.SKIPPED),
    step('jingle_unittests', result=results.SKIPPED),
    step('device_unittests', result=results.SKIPPED),
    step('media_unittests', result=results.SKIPPED),
    step('net_unittests', result=results.SKIPPED),
    step('ppapi_unittests', result=results.SKIPPED),
    step('printing_unittests', result=results.SKIPPED),
    step('printing_unittests_buildrunner_ignore', hidden=True),
    step('remoting_unittests', result=results.SKIPPED),
    step('test_shell_tests', result=results.SKIPPED),
    step('ipc_tests', result=results.SKIPPED),
    step('unit_tests', result=results.SKIPPED),
    step('sql_unittests', result=results.SKIPPED),
    step('ui_unittests', result=results.SKIPPED),
    step('content_unittests', result=results.SKIPPED),
    step('views_unittests', result=results.SKIPPED),
    step('browser_tests', result=results.SKIPPED),
    step('chromedriver_tests', result=results.SKIPPED),
    step('content_browsertests', result=results.SKIPPED),
    step('installer_util_unittests', result=results.SKIPPED),
    step('sizes', result=results.SKIPPED),
    step('sync_integration_tests'),
    step('nacl_integration', result=results.SKIPPED),
    step('process_dumps'),
  ],
  'master': {
    'getTitle()': 'Title',
    'getBuildbotURL()': 'http://build.chromium.org/p/tryserver.chromium/',
  },
}
