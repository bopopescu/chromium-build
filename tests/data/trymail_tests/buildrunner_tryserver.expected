From nobody Wed Dec 31 16:00:01 1969
Content-Type: multipart/mixed; boundary="===============7454617213454723890=="
MIME-Version: 1.0
Date: Wed, 31 Dec 1969 16:00:01 -0800
Subject: try success for 11941002-33001: Rebase on win_rel @ r178097
From: from@example.org

--===============7454617213454723890==
Content-Type: text/html; charset="iso-8859-1"
MIME-Version: 1.0
Content-Transfer-Encoding: quoted-printable

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html xmlns=3D"http://www.w3.org/1999/xhtml">
<head>
  <title>try success for 11941002-33001: Rebase on win_rel @ r178097</title>
</head>
<body style=3D"font-family: Verdana, Cursor; font-size: 10px;">
  try success for 11941002-33001: Rebase on win_rel @ r178097<p>
  <a href=3D"http://build.chromium.org/p/tryserver.chromium/">http://build.=
chromium.org/p/tryserver.chromium/</a><p>
  You are awesome! Try succeeded!<p> =

  Build: <a href=3D"http://build.chromium.org/p/tryserver.chromium/builders=
/win_rel/builds/102110">102110</a> on
         <a href=3D"http://build.chromium.org/p/tryserver.chromium/builders=
/win_rel">win_rel</a><br>
  slave: <a href=3D"http://build.chromium.org/p/tryserver.chromium/buildsla=
ves/vm355-m4">vm355-m4</a><br>
    issue: <a href=3D"https://chromiumcodereview.appspot.com/11941002#ps330=
01">11941002</a><br>raw patchset: <a href=3D"https://chromiumcodereview.app=
spot.com/download/issue11941002_33001.diff">33001</a><br><table style=3D"bo=
rder-spacing: 1px 1px; font-weight: bold; padding: 3px 0px 3px 0px; text-al=
ign: center; font-size: 10px; font-family: Verdana, Cursor; ">
<tr><td style=3D"color: #FFFFFF; background-color: #8fdf5f; border-color: #=
4F8530;"><a title=3D"Reason: 11941002-33001: Rebase" href=3D"http://build.c=
hromium.org/p/tryserver.chromium/builders/win_rel/builds/102110">Build 1021=
10</a></td></tr><tr><td style=3D"color: #FFFFFF; background-color: #8fdf5f;=
 border-color: #4F8530;">svnkill<br/><a href=3D"http://build.chromium.org/p=
/tryserver.chromium/builders/win_rel/builds/102110/steps/svnkill/logs/stdio=
">stdio</a></td></tr><tr><td style=3D"color: #FFFFFF; background-color: #8f=
df5f; border-color: #4F8530;">update_scripts<br/><a href=3D"http://build.ch=
romium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/update=
_scripts/logs/stdio">stdio</a></td></tr><tr><td style=3D"color: #FFFFFF; ba=
ckground-color: #8fdf5f; border-color: #4F8530;">taskkill<br/><a href=3D"ht=
tp://build.chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110=
/steps/taskkill/logs/stdio">stdio</a></td></tr><tr><td style=3D"color: #FFF=
FFF; background-color: #8fdf5f; border-color: #4F8530;">update<br/>r178097<=
br/>webkit r140218<br/>nacl r10619<br/>v8 r13451<br/><a href=3D"http://buil=
d.chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/up=
date/logs/stdio">stdio</a></td></tr><tr><td style=3D"color: #FFFFFF; backgr=
ound-color: #8fdf5f; border-color: #4F8530;">apply_issue<br/><a href=3D"htt=
p://build.chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110/=
steps/apply_issue/logs/stdio">stdio</a></td></tr><tr><td style=3D"color: #F=
FFFFF; background-color: #8fdf5f; border-color: #4F8530;">gclient hooks<br/=
><a href=3D"http://build.chromium.org/p/tryserver.chromium/builders/win_rel=
/builds/102110/steps/runhooks/logs/stdio">stdio</a></td></tr><tr><td style=
=3D"color: #FFFFFF; background-color: #8fdf5f; border-color: #4F8530;">clea=
nup_temp<br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/buil=
ders/win_rel/builds/102110/steps/cleanup_temp/logs/stdio">stdio</a></td></t=
r><tr><td style=3D"color: #FFFFFF; background-color: #ffc343; border-color:=
 #C29D46;">compile<br/><a href=3D"http://build.chromium.org/p/tryserver.chr=
omium/builders/win_rel/builds/102110/steps/compile/logs/stdio">stdio</a></t=
d></tr><tr><td style=3D"">manifest_to_hashes<br/><a href=3D"http://build.ch=
romium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/manife=
st_to_hashes/logs/stdio">stdio</a></td></tr><tr><td style=3D"">trigger_swar=
m_triggered<br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/b=
uilders/win_rel/builds/102110/steps/trigger_swarm_triggered/logs/stdio">std=
io</a></td></tr><tr><td style=3D"color: #FFFFFF; background-color: #8fdf5f;=
 border-color: #4F8530;">start_crash_handler<br/><a href=3D"http://build.ch=
romium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/start_=
crash_handler/logs/stdio">stdio</a></td></tr><tr><td style=3D"color: #FFFFF=
F; background-color: #8fdf5f; border-color: #4F8530;">interactive_ui_tests<=
br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/builders/win_=
rel/builds/102110/steps/interactive_ui_tests/logs/stdio">stdio</a></td></tr=
><tr><td style=3D"">check_deps2git<br/><a href=3D"http://build.chromium.org=
/p/tryserver.chromium/builders/win_rel/builds/102110/steps/check_deps2git/l=
ogs/stdio">stdio</a></td></tr><tr><td style=3D"">check_deps2submodules<br/>=
<a href=3D"http://build.chromium.org/p/tryserver.chromium/builders/win_rel/=
builds/102110/steps/check_deps2submodules/logs/stdio">stdio</a></td></tr><t=
r><td style=3D"">check_deps<br/><a href=3D"http://build.chromium.org/p/trys=
erver.chromium/builders/win_rel/builds/102110/steps/check_deps/logs/stdio">=
stdio</a></td></tr><tr><td style=3D"">base_unittests<br/><a href=3D"http://=
build.chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110/step=
s/base_unittests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">cacheinv=
alidation_unittests<br/><a href=3D"http://build.chromium.org/p/tryserver.ch=
romium/builders/win_rel/builds/102110/steps/cacheinvalidation_unittests/log=
s/stdio">stdio</a></td></tr><tr><td style=3D"">cc_unittests<br/><a href=3D"=
http://build.chromium.org/p/tryserver.chromium/builders/win_rel/builds/1021=
10/steps/cc_unittests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">chr=
omedriver_unittests<br/><a href=3D"http://build.chromium.org/p/tryserver.ch=
romium/builders/win_rel/builds/102110/steps/chromedriver_unittests/logs/std=
io">stdio</a></td></tr><tr><td style=3D"">components_unittests<br/><a href=
=3D"http://build.chromium.org/p/tryserver.chromium/builders/win_rel/builds/=
102110/steps/components_unittests/logs/stdio">stdio</a></td></tr><tr><td st=
yle=3D"">courgette_unittests<br/><a href=3D"http://build.chromium.org/p/try=
server.chromium/builders/win_rel/builds/102110/steps/courgette_unittests/lo=
gs/stdio">stdio</a></td></tr><tr><td style=3D"">crypto_unittests<br/><a hre=
f=3D"http://build.chromium.org/p/tryserver.chromium/builders/win_rel/builds=
/102110/steps/crypto_unittests/logs/stdio">stdio</a></td></tr><tr><td style=
=3D"">url_unittests<br/><a href=3D"http://build.chromium.org/p/tryserver.ch=
romium/builders/win_rel/builds/102110/steps/url_unittests/logs/stdio">stdio=
</a></td></tr><tr><td style=3D"">gpu_unittests<br/><a href=3D"http://build.=
chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/gpu_=
unittests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">jingle_unittest=
s<br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/builders/wi=
n_rel/builds/102110/steps/jingle_unittests/logs/stdio">stdio</a></td></tr><=
tr><td style=3D"">device_unittests<br/><a href=3D"http://build.chromium.org=
/p/tryserver.chromium/builders/win_rel/builds/102110/steps/device_unittests=
/logs/stdio">stdio</a></td></tr><tr><td style=3D"">media_unittests<br/><a h=
ref=3D"http://build.chromium.org/p/tryserver.chromium/builders/win_rel/buil=
ds/102110/steps/media_unittests/logs/stdio">stdio</a></td></tr><tr><td styl=
e=3D"">net_unittests<br/><a href=3D"http://build.chromium.org/p/tryserver.c=
hromium/builders/win_rel/builds/102110/steps/net_unittests/logs/stdio">stdi=
o</a></td></tr><tr><td style=3D"">ppapi_unittests<br/><a href=3D"http://bui=
ld.chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/p=
papi_unittests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">printing_u=
nittests<br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/buil=
ders/win_rel/builds/102110/steps/printing_unittests/logs/stdio">stdio</a></=
td></tr><tr><td style=3D"">remoting_unittests<br/><a href=3D"http://build.c=
hromium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/remot=
ing_unittests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">test_shell_=
tests<br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/builder=
s/win_rel/builds/102110/steps/test_shell_tests/logs/stdio">stdio</a></td></=
tr><tr><td style=3D"">ipc_tests<br/><a href=3D"http://build.chromium.org/p/=
tryserver.chromium/builders/win_rel/builds/102110/steps/ipc_tests/logs/stdi=
o">stdio</a></td></tr><tr><td style=3D"">unit_tests<br/><a href=3D"http://b=
uild.chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps=
/unit_tests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">sql_unittests=
<br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/builders/win=
_rel/builds/102110/steps/sql_unittests/logs/stdio">stdio</a></td></tr><tr><=
td style=3D"">ui_unittests<br/><a href=3D"http://build.chromium.org/p/tryse=
rver.chromium/builders/win_rel/builds/102110/steps/ui_unittests/logs/stdio"=
>stdio</a></td></tr><tr><td style=3D"">content_unittests<br/><a href=3D"htt=
p://build.chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110/=
steps/content_unittests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">v=
iews_unittests<br/><a href=3D"http://build.chromium.org/p/tryserver.chromiu=
m/builders/win_rel/builds/102110/steps/views_unittests/logs/stdio">stdio</a=
></td></tr><tr><td style=3D"">browser_tests<br/><a href=3D"http://build.chr=
omium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/browser=
_tests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">chromedriver_tests=
<br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/builders/win=
_rel/builds/102110/steps/chromedriver_tests/logs/stdio">stdio</a></td></tr>=
<tr><td style=3D"">content_browsertests<br/><a href=3D"http://build.chromiu=
m.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/content_bro=
wsertests/logs/stdio">stdio</a></td></tr><tr><td style=3D"">installer_util_=
unittests<br/><a href=3D"http://build.chromium.org/p/tryserver.chromium/bui=
lders/win_rel/builds/102110/steps/installer_util_unittests/logs/stdio">stdi=
o</a></td></tr><tr><td style=3D"">sizes<br/><a href=3D"http://build.chromiu=
m.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/sizes/logs/=
stdio">stdio</a></td></tr><tr><td style=3D"color: #FFFFFF; background-color=
: #8fdf5f; border-color: #4F8530;">sync_integration_tests<br/><a href=3D"ht=
tp://build.chromium.org/p/tryserver.chromium/builders/win_rel/builds/102110=
/steps/sync_integration_tests/logs/stdio">stdio</a></td></tr><tr><td style=
=3D"">nacl_integration<br/><a href=3D"http://build.chromium.org/p/tryserver=
.chromium/builders/win_rel/builds/102110/steps/nacl_integration/logs/stdio"=
>stdio</a></td></tr><tr><td style=3D"color: #FFFFFF; background-color: #8fd=
f5f; border-color: #4F8530;">process_dumps<br/><a href=3D"http://build.chro=
mium.org/p/tryserver.chromium/builders/win_rel/builds/102110/steps/process_=
dumps/logs/stdio">stdio</a></td></tr></table>
<br>
FAQ: <a href=3D"https://sites.google.com/a/chromium.org/dev/developers/test=
ing/try-server-usage">
https://sites.google.com/a/chromium.org/dev/developers/testing/try-server-u=
sage</a><br>
</body>
</html>

--===============7454617213454723890==--